src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -159,6 +159,14 @@ private static String convertElementNode(Node currentNode, String indentation, b
                     return generateStudentMotdPlaceholder(indentation);
                 }
             }
+        } else if (currentNode.getNodeName().equalsIgnoreCase("style")) {
+            NamedNodeMap attributes = currentNode.getAttributes();
+            for (int i = 0; i < attributes.getLength(); i++) {
+                Node attribute = attributes.item(i);
+                if (isTinymceStyleAttribute(attribute)) {
+                    return generateTinymceStylePlaceholder(indentation);
```

Comment:
The auto-generated style differs depending on the browser, so at best what I can do is replacing with placeholder.